XRTK.Oculus/Packages/com.xrtk.oculus/Controllers/BaseOculusController.cs

```diff
@@ -37,48 +37,54 @@ public class BaseOculusController : BaseController
             NodeType = nodeType;
         }
 
-        public OculusApi.Node NodeType { get; } = OculusApi.Node.None;
+        /// <summary>
+        /// The Oculus Note Type.
```

Comment:
Node type, not note type.

Plus, can you please reinstate the initializer, as in original testing it causes errors on first run without them (oculus)

Reply:
The initializer is redundant. The constructor should set the default value (which is the same)